compress/README.md

```diff
@@ -14,11 +14,14 @@ type Options struct {
 	CleanFonts                      bool
 	SubsetFonts                     bool
 	CleanContentstream              bool
+	CleanUnusedResources            bool
 }
 ```
 
+From the available filters `CleanFonts`, `CleanUnusedResources`, `CombineDuplicateStreams`, `CombineDuplicateDirectObjects`,`CombineDuplicateDirectObjects`, `CleanFonts` and `CompressStreams` enable lossless compressions.

```

Comment:
UseObjectStreams also
CombineIdenticalIndirectObjects also

Actually most of them are lossless except ImageUpperPPI and ImageQuality.